geonode/harvesting/admin.py

```diff
@@ -157,6 +158,10 @@ def update_harvestable_resources(self, request, queryset):
                 f"Updating harvestable resources asynchronously for {being_updated}..."
             )
 
+    @admin.display(description="Number of available resources to harvest")
+    def get_numb_available_resources(self, harvester: models.Harvester):
+        return harvester.num_available_resources
+
```

Comment:
Please do not introduce another qualifier for resources.

The concept of `harvestable resources` is enough, I don't think we need to also have `available resources`.

Reply:
updated @ricardogsilva

Reply:
updated @ricardogsilva